families/families.js

```diff
@@ -4,41 +4,51 @@ checkAuth();
 
 const familiesEl = document.querySelector('.families-container');
 const logoutButton = document.getElementById('logout');
-
+console.log(familiesEl);
 logoutButton.addEventListener('click', () => {
     logout();
 });
 
-function displayFamilies() {
-    // fetch families from supabase
+async function displayFamily() {
```

Comment:
On line 34, it looks like you call displayFamily with a parameter of newFamilies, remember to write your function as it is used throughout your code. In this case, displayFamily should have a parameter of families.